examples/rbfe_single.py

```diff
@@ -44,6 +45,7 @@ def wrap_method(args: Iterable[Any], fxn: callable):
         "--num_gpus",
         type=int,
         help="number of gpus",
+        default=get_gpu_count(),
         required=True
```

Comment:
How does required=True work with default values?

Reply:
It still prompts you, so should probably remove the required=True